resource-managers/yarn/src/main/scala/org/apache/spark/deploy/yarn/ApplicationMaster.scala

```diff
@@ -836,6 +833,20 @@ object ApplicationMaster extends Logging {
 
 }
 
+object ApplicationMasterUtil {
```

Comment:
Just add this to the `ApplicationMaster` object instead of creating yet another utils object... and also add `private[spark]` to the method there.